src/microbe_stage/MicrobeAI.cs

```diff
@@ -367,6 +367,14 @@ private void EngagePrey(Vector3 target, Random random, bool engulf)
     // For doing run and tumble
     private void RunAndTumble(Random random)
     {
+        // If this microbe is actually immobile, just initialize by moving in a random direction.
+        // Used to get newly spawned microbes to move.
+        if (microbe.MovementDirection.Length() == 0)
+        {
+            MoveWithRandomTurn(0, 2 * Mathf.Pi, random);
```

Comment:
```suggestion
            MoveWithRandomTurn(0, Mathf.Pi, random);
```

MoveWithRandomTurn picks a direction and turns relative to the current heading, so just pi should be enough to be able to turn any direction.